### PR TITLE
Fixes variation theme dialogue box behaviour.

### PR DESCRIPTION
**Pull request by anantanant2015**
## Why?

- Warning dialogue for variation should only come when product has no variants.
- Adding or changing the variation theme dialogue box should display no button, 
  and message displayed should also be corrected to "Changing the variation 
  theme would remove all the existing variants, are you sure you want to proceed?"

## This change addresses the need by:

- Because warning dialogue was displayed even if the product has no variant.
- Because no button in the dialogue box was displayed on mousehover, and the 
  display message was not giving correct details.

[delivers #162545068]

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

### CODE REVIEW

**Comment by ashish173** [comment]
@anantanant2015 you have not fully understood the ticket. It clearly says that we can't show this if there are not existing variants with a product. Make the specified changes. Please reach out to me if you have any more confusions.

**Comment by anantanant2015** [comment]
@ashish173 bug fixed

File: apps/admin_app/assets/js/views/product/edit.js [hunk 1]
```diff
@@ -163,22 +163,17 @@ function setupProduct() {
 
   // This handle the variation theme selection
   $('#product_theme_id').on('change', function (e) {
-    $('#theme_change_modal').modal(`show`);
+    var product_variants = document.getElementsByClassName("product_has_variants");
```
Comment (ashish173) [inline apps/admin_app/assets/js/views/product/edit.js]: leave an empty line in between.

File: apps/admin_app/assets/js/views/product/edit.js [hunk 2]
```diff
@@ -229,3 +224,18 @@ function update_option_value(id, value){
...
+
+function create_update_variation_theme() {
+  var optionSelected = $('#product_theme_id');
+    var valueSelected = optionSelected.val();
+    const product_id = optionSelected
+      .parents()
+      .find('#product_id');
+    const new_variant = optionSelected
+      .parents()
+      .find('#new_variant');
```
Comment (ashish173) [inline apps/admin_app/assets/js/views/product/edit.js]: leave a line.

**Pull request merged**